src/utils/logging-buffer.ts

```diff
@@ -88,24 +107,9 @@ export class LogBuffer {
     this.buffer = [];
 
     try {
-      const response = await fetch(`${this.acrolinxUrl}/int-service/api/v1/logs`, {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify({
-          appName: 'sidebar-client-app',
-          logs: logsToSend,
-        }),
-      });
-
-      if (response.ok) {
-        console.log('Logs successfully sent to the server');
-        this.retries = 0;
-      } else {
-        console.error('Failed to send logs to the server');
-        this.handleRetry(logsToSend);
-      }
+      await this.intService.sendLogs(this.appName, logsToSend, this.accessToken);
+      console.log('Logs successfully sent to the server');
+      this.retries = 0;
     } catch (error) {
       console.error('Error sending logs to the server:', error);
```

Comment:
If retries fail then only error make sense
Question is why retries will work if error is like 4XX.

Also what handletries does - is it to re submitting the request or just clean the buffer?

Reply:
hmmm, thats a very valid point. I'm adjusting it to retry only for valid error codes;

handleRetry is re-submitting the request, not just cleaning the buffer.